nixos/infrastructure/testing.nix

```diff
@@ -18,11 +18,19 @@
     services.openssh.enable = lib.mkOverride 60 false;
     # build-vms.nix from NixOS automatically generates numbered interface
     # configs with default IPs. We rename the devices to fe and srv early so
-    # the services wait for eth1 and eth2 for 5 minutes and time out.
+    # the services wait for the interfaces for 5 minutes and time out.
```

Comment:
I like this! Can we extract that into a separate PR, please?

Reply:
Has been split https://github.com/flyingcircusio/fc-nixos/pull/372

Reply:
Note that I'd need that other PR merged before this one then

Reply:
Done